otellogr: Fix nil context panic (#6527)
open-telemetry/opentelemetry-go-contrib: CHANGELOG.md

- Generate server metrics with semantic conventions v1.26 in `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp` when `OTEL_SEMCONV_STABILITY_OPT_IN` is set to `http/dup`. (#6411)

### Fixed

> - Fixed an issue where otellogr emits nil context which leads to panic in exporters `go.opentelemetry.io/contrib/bridges/otellogr`. (#6527)

Reviewer: ```suggestion
- Use `context.Background()` as default context instead of nil in `go.opentelemetry.io/contrib/bridges/otellogr`. (#6527)
```
